client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/database/schema/ColumnDto.java

```diff
@@ -22,4 +22,10 @@
 
     public boolean isUnsigned = false;
 
+    /**
+     * precision of number
+     * -1 means that the precision is unspecified or inapplicable
+     */
```

Comment:
do not check for specific values. have something more general like `a negative number (e.g., -1) means...`